### PR TITLE
[res/tflite] Added recipe for YUV to RGB

### PR DESCRIPTION
**Pull request by AShedko**
Added recipe for YUV to RGB
Signed-off-by: Andrey Shedko <a.shedko@samsung.com>

### CODE REVIEW

File: res/TensorFlowLiteRecipes/YUV_TO_RGB_U8_000/test.recipe [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    new_shape: 1
+    new_shape: 4
+    new_shape: 4
+    new_shape: 3
+  }
+}
+
+input: "input"
+input: "weight"
+input: "bias"
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/YUV_TO_RGB_U8_000/test.recipe:66]: I think weight and bias should not be in input of the network

Comment (AShedko) [reply res/TensorFlowLiteRecipes/YUV_TO_RGB_U8_000/test.recipe:66]: See `FullyConnected_001` for reference
https://github.com/Samsung/ONE/blob/9b2a77d442a12fd89bad280d7002013946c1b95d/res/TensorFlowLiteRecipes/FullyConnected_001/test.recipe#L30-L34

Comment (seanshpark) [reply res/TensorFlowLiteRecipes/YUV_TO_RGB_U8_000/test.recipe:66]: Um.. there is not strict one but here is my understanding of this recipe.

`FullyConnected_001 ` is to test `FullyConnected_001 ` with inputs weight and bias. For FC, weight and bias can be any thing as it doesn't mater where or what the values are. we are just creating `FullyConnected`

For YUV_to_RGB, our purpose is to test `out = yuv_to_rgb(input)` sematically. there is no weight or bias as input. And `FullyConnected` is happend to be used as operator.

Comment (seanshpark) [reply res/TensorFlowLiteRecipes/YUV_TO_RGB_U8_000/test.recipe:66]: @binarman , can you give your opinion about this?

Comment (binarman) [reply res/TensorFlowLiteRecipes/YUV_TO_RGB_U8_000/test.recipe:66]: @seanshpark 
I agree with you.
At some point we may even try to folds this into circle-specific operator. (for example, for better quantization)
So we need to be more specific in this case.

P.S. @AShedko is absent today and probably tomorrow, so I'll add weights myself

**Comment by binarman** [comment]
Note:
I got weights for quantized model from code (tf 2.0.0)
```python
import tensorflow as tf
import numpy as np
root = tf.train.Checkpoint()
root.f = tf.function(lambda x: tf.image.yuv_to_rgb(x))
input_x = np.ones(shape=[4, 4, 3], dtype=np.float32)
concrete_func = root.f.get_concrete_function(input_x)
def representative_dataset_gen():
  base_layer = np.ones((4, 4, 1), dtype=np.float32)
  y_low = base_layer * 0
  y_high = base_layer
  uv_low = base_layer * -0.5;
  uv_high = base_layer * 0.5;
  yield [np.concatenate((y_low, uv_low, uv_low), axis=2)]
  yield [np.concatenate((y_high, uv_high, uv_high), axis=2)]
converter = tf.lite.TFLiteConverter.from_concrete_functions([concrete_func])
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.representative_dataset = representative_dataset_gen
tflite_model = converter.convert()
with open("model.tflite", "wb") as f:
    f.write(tflite_model)
``` 

Though, not sure we will use this model, maybe separate quantized operator will have better accuracy.

**Comment by binarman** [comment]
@lucenticus 
Maybe you will be interested in quantized model.
